templates/data_assessment.html

```diff
@@ -0,0 +1,475 @@
+            <div class="error-message" style="display:none;"></div>
+        </div>
+
+        <!-- Question 14 -->
+        <div class="question" id="q14">
+            <label class="question-label">14. How much trust is there in the data across your organization?</label>
+            <div class="options">
+                <input type="radio" id="q14a" name="q14" value="high-trust">
+                <label class="answer-option-label" for="q14a">High trust: data quality is consistently good, and people rely on it for decision-making</label><br>
+                <input type="radio" id="q14b" name="q14" value="moderate-trust">
+                <label class="answer-option-label" for="q14b">Moderate trust: occasional issues with data quality, but mostly reliable</label><br>
+                <input type="radio" id="q14c" name="q14" value="low-trust">
+                <label class="answer-option-label" for="q14c">Low trust: frequent issues with data quality, leading to conflicting insights</label><br>
+                <input type="radio" id="q14d" name="q14" value="no-trust">
+                <label class="answer-option-label" for="q14d">No trust: data quality is poor, resulting in decisions being made without data</label>
+            </div>
+            <textarea name="q14_text" placeholder="Optional: Provide additional details"></textarea>
+            <div class="error-message" style="display:none;"></div>
+        </div>
+
+        <!-- Question 15 -->
+        <div class="question" id="q15">
+            <label class="question-label">15. What level of standardization do you have for metrics and KPIs?</label>
+            <div class="options">
+                <input type="radio" id="q15a" name="q15" value="not-standardized">
+                <label class="answer-option-label" for="q15a">Metrics are not standardized, leading to differing conclusions</label><br>
+                <input type="radio" id="q15b" name="q15" value="fully-standardized">
+                <label class="answer-option-label" for="q15b">Metrics and KPIs are fully standardized and agreed upon by all teams</label><br>
+                <input type="radio" id="q15c" name="q15" value="partially-standardized">
+                <label class="answer-option-label" for="q15c">Some metrics are standardized, but there are discrepancies between teams</label><br>
+                <input type="radio" id="q15d" name="q15" value="no-defined-metrics">
+                <label class="answer-option-label" for="q15d">We don’t currently have any defined metrics or KPIs</label>
+            </div>
+            <textarea name="q15_text" placeholder="Optional: Provide additional details"></textarea>
+            <div class="error-message" style="display:none;"></div>
+        </div>
+
+        <!-- Navigation Buttons -->
+        <div class="submit-container">
+            <button type="button" id="prev-btn" style="display:none;">Previous</button>
```

Comment:
```suggestion
           <button type="button" id="start-btn">Start</button>
```

Create a unique start button to begin that way you can push it to the right side
you've got two options with CSS -- I'd suggest the first:
1. `flex-direction: row-reverse;`
2. `float: right;`